Policies/github-privacy-statement.md

```diff
@@ -24,7 +25,7 @@ All capitalized terms have their definition in [GitHub’s Terms of Service](/gi
 
 ## The short version
 
-We use your personal information as this Privacy Statement describes. No matter where you are, where you live, or what your citizenship is, we provide the same high standard of privacy protection to all our users around the world, regardless of their country of origin or location.
+We use your personal information as this Privacy Statement describes. No matter where you are, where you live, or what your citizenship is, you have the same high standard of privacy protection when using GitHub's products as all our users around the world, regardless of their country of origin or location.
```

Comment:
I don't quite understand why this statement of what is provided was replaced by what appears to be a guarantee that something is already possessed. It seems to me like it would incur unnecessary legal liability if the user were to somehow not receive this, whereas the previous version merely stated that it was provided, not even whom to.